cgi-bin/sitemap.js

```diff
@@ -10,12 +10,61 @@
  * governing permissions and limitations under the License.
  */
 const algoliasearch = require('algoliasearch');
+const pick = require('lodash.pick');
+const rp = require('request-promise-native');
```

Comment:
Better use `helix-fetch`

Reply:
or use the downloader in helix pipeline which is already available in helix-pages